### ISSUE TITLE
Switch between NLU models in Rasa core as we do in rasa nlu.

### ISSUE THREAD

**Issue by neumann-d** [RasaHQ/rasa_nlu#3135]
Similar to #2874 I want to ask, if it is possible to optionally pass a nlu model parameter to rasa core during a request via HTTP API.

In the docs, I can only find the option to set the nlu model in rasa core during the start with the `-u` parameter:
```
$ python -m rasa_core.run \
    --enable_api \
    -d models/dialogue \
    -u models/nlu/current \
    -o out.log
```

But in rasa nlu you can change the model for each request:
```
$ curl -XPOST localhost:5000/parse -d '{"q":"hello there", "model": "<model_XXXXXX>"}'
```

Would be nice to have this flexibility in the rasa core HTTP API too.

**Comment by akelad** [comment]
Thanks for raising this issue, @ricwo  will get back to you about it soon.

**Comment by ricwo** [comment]
@neumann-d this is currently not possible, as the Rasa Core only knows about a single NLU model at a time (it does not have a DataRouter or ProjectStore - unlike the NLU server). This functionality might become part of future releases though

**Comment by akelad** [comment]
closing, because we're keeping track of that in a different issue

**Comment by neumann-d** [comment]
Thank you for the response and this awesome project!

**Comment by neumann-d** [comment]
Seems to be solved since Rasa 1.0. Great work guys!
